Items/Weapons/TopazStaff.cs

```diff
@@ -3,7 +3,7 @@
 using System.Collections.Generic;
 using Terraria;
 using Terraria.ID;
-using Terraria.Localization;
+using Terraria.GameContent.Creative;
```

Comment:
Did you forget to make this researchable?

Reply:
Yes, yes I did. fixed in the next commit.